### ISSUE TITLE
feature request: unknown or out of scope detection

### ISSUE THREAD

**Issue by DanielCollins** [kaunghtetsan275/pyidaungsu#4]
for example

    pds.detect("i like turtles")
    
is detected as "shan".

Of course you cannot detect every possible language in the world, but "shan" is obviously wrong here, so it would be useful if it could return `None` or `"unknown"` in cases where there is no accurate match.

**Comment by DanielCollins** [comment]
my usecase is tokenizing in a burmese+english bilingual project. for english, i switch to spacy.

for detecting whether to use pyidaungsu or spacy, i currently just use both, and take the result of whichever one split into a larger number of tokens. since pyidaungsu doesn't split english into any tokens at all, that actually works totally fine. but it feels like a hack.

**Comment by kaunghtetsan275** [comment]
> Of course you cannot detect every possible language in the world, but "shan" is obviously wrong here, so it would be useful if it could return `None` or `"unknown"` in cases where there is no accurate match.

The language detection model will always give a result of something even if it misidentifies the language, so it isn't possible to add unknown tag. However, English can be added as a supported language which I did in the new 0.1.3 version. Simply install again to get the detection for new languages.

`pip install -U pyidaungsu`

This library is initially intended for local researchers so I forgot to add more popular global languages. I've added new languages and as of now the version 0.1.3 supports English, Spanish, French, Chinese, Korean, Japanese, Burmese (unicode), Burmese (zawgyi) and Karen.

Shan and Mon languages are dropped in this version detection since the datasets I've used weren't clean and consist of many mixed languages. I will add those languages again for detection in the next releases. Tokenization for these languages can still be used though.

> since pyidaungsu doesn't split english into any tokens at all, that actually works totally fine. but it feels like a hack.

Pyidaungsu doesn't support Stemming and Lemmatization for English yet so I suggest you keep using SpaCy or NLTK if you want those features. Pyidaungsu can only support space-separated, word-level tokenization for English for now.

Currently, language detection for single word doesn't perform really well.
For example, it will misidentify "turtle" as French.
But "I like turtles" will correctly be identified as English. So the input as a sentence instead of a single word will be more appropriate for correct detection.
